src/components/workResume.svelte

```diff
@@ -1,18 +1,32 @@
 <script lang="ts">
   import type { Highlight } from '@src/data/data';
   import type { Writable } from 'svelte/store';
+  import { tagsStore, Tag } from '@src/data/tag';
 
   export let name: Writable<string>;
   export let position: Writable<string>;
   export let startDate: Writable<string>;
   export let endDate: Writable<string>;
   export let highlights: Writable<Array<Highlight>>;
+
+  export function isHighlightVisible(h: Highlight, tags: Tag[]): boolean {
```

Comment:
I've already defined this in `resumeUtils`, so I should make sure I'm using the right function